exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricConfiguration.java

```diff
@@ -45,6 +46,9 @@ public abstract class MetricConfiguration {
   /** Resource attribute filter that disables addition of resource attributes to metric labels. */
   public static final Predicate<AttributeKey<?>> NO_RESOURCE_ATTRIBUTES = attributeKey -> false;
 
+  public static final MonitoredResourceDescription DEFAULT_MONITORED_RESOURCE_DESCRIPTION =
```

Comment:
nit: should we call this "empty monitored resource description"?  It is also the default, but its probably more important that it is empty or the zero state.